<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>398</width>
      <height>30</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>120</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-color:rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="placeholderText">
     <string>Search Twitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>searchIcon.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>8</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>29</red>
          <green>155</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_search{
background-color:rgb(29,155,240);
border-radius:15px;
}

QPushButton#pushButton_search:hover {
    background-color: rgb(26,140,216);
    border-radius:15px;
}
</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>59</y>
     <width>643</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tweets">
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <attribute name="title">
     <string>Tweets</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_tweets">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>637</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QScrollBar:vertical {
	background-color: rgb(206, 206, 206);
    width: 14px;
    margin: 15px 0 15px 0;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(206, 206, 206);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{	
	background-color: rgb(193, 193, 193);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::sub-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::add-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>1022</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"> QScrollBar:vertical {
	background-color: rgb(206, 206, 206);
    width: 14px;
    margin: 15px 0 15px 0;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(206, 206, 206);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{	
	background-color: rgb(193, 193, 193);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::sub-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::add-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>1000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>601</width>
            <height>971</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_tweets"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="people">
    <attribute name="title">
     <string>People</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_people">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>637</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QScrollBar:vertical {
	background-color: rgb(206, 206, 206);
    width: 14px;
    margin: 15px 0 15px 0;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(206, 206, 206);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{	
	background-color: rgb(193, 193, 193);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::sub-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::add-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>1022</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"> QScrollBar:vertical {
	background-color: rgb(206, 206, 206);
    width: 14px;
    margin: 15px 0 15px 0;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(206, 206, 206);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{	
	background-color: rgb(193, 193, 193);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::sub-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}

QScrollBar::add-line:vertical {
	background-color: rgb(206, 206, 206);
	height: 15px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover {
	background-color: rgb(193, 193, 193);
}

QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(165, 165, 165);
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>1000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>601</width>
            <height>211</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_people">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetNoConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
